packages/frontend/src/providers/DashboardProvider.tsx

```diff
@@ -55,6 +60,9 @@ type DashboardContext = {
     haveFiltersChanged: boolean;
     setHaveFiltersChanged: Dispatch<SetStateAction<boolean>>;
     addSuggestions: (newSuggestionsMap: Record<string, string[]>) => void;
+    allFilterableFields: FilterableField[] | undefined;
+    filterableFieldsBySavedQueryUuid: DashboardAvailableFilters | undefined;
+    filterableFieldsByTileUuid: DashboardAvailableFilters | undefined;
```

Comment:
we can get rid of `filterableFieldsByTileUuid` if we rewrite "dashboard tile filters" components a bit but let's not do that in this PR.